src/components/SideBar/SideBar.js

```diff
@@ -0,0 +1,150 @@
+export const SideBar = ({ selectTab }) => {
+  return (
+    <div>
+      <div
+        style={{
+          fontSize: "0.7rem",
+          fontWeight: "700",
+          opacity: "0.9",
+          marginTop: "1.5rem",
+          marginLeft: "3rem",
+          color: "#023e8a",
+          padding: "0.5rem 0rem"
+        }}
+      >
+        GETTING STARTED
```

Comment:
Same as above, consider using css property `text-transform: uppercase;` for such operations.